MetaMask/metamask-extension
# blockaid settings.

## Explanation

As we introduce the Blockaid security provider feature we should add a toggle under experimental to allow users to turn the feature on and off.

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/527


## Screenshots/Screencaps

<img width="355" alt="Screenshot 2023-07-08 at 15 43 02" src="https://github.com/MetaMask/metamask-extension/assets/44811/c02fb50f-9c70-4f89-a6af-ab0a7bf35f44">

## Manual Testing Steps

1. Open Settings
2. Tab Experimental
3. You should see the screenshot above
4. Toggle button should be off by default
5. Tap on the toggle button it should stay on
6. Restart MetaMask
7. Toggle should remain on
8. Repeat steps 5-7, this time make sure toggle stays off

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
